CirrusMDSDK.framework/Headers/CirrusMDSDK-Swift.h

```diff
@@ -213,10 +220,13 @@ SWIFT_CLASS_PROPERTY(@property (nonatomic, class, readonly, strong) CirrusMDSDKS
 
 
 
+
+
 @class UIViewController;
 enum CirrusMDSDKSessionResult : NSInteger;
 
 @interface CirrusMDSDKSession (SWIFT_EXTENSION(CirrusMDSDK))
+- (void)setConfig:(CirrusMDSDKConfig * _Nonnull)config;
```

Comment:
What's this for? How is this changing but there's no tests?

Reply:
That file is generated by the Swift build system.